cirq-superstaq/cirq_superstaq/service_test.py

```diff
@@ -503,7 +505,7 @@ def test_service_ibmq_compile(mock_ibmq_compile: mock.MagicMock) -> None:
         "final_logical_to_physicals": cirq.to_json([list(final_logical_to_physical.items())]),
     }
 
-    assert service.ibmq_compile(circuit).circuit == circuit
+    assert service.ibmq_compile(circuit, options={"test": "yes"}).circuit == circuit
```

Comment:
off topic but we should definitely have a function validating that included options are supported.

Reply:
Agreed, this should definitely happen. I am not sure if we should be talking about the private repo's code in this public repo though. (not sure what the policy is here)